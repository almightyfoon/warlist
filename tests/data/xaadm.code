c4101b_-0Vkt1s1skpkr2Ikq4ikgkg1_kk32ko___-0B0Fd-1h1h2Ai22E7y2oi5i5bBbB
