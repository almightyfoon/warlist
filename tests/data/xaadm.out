https://conflictchamber.com/#c4101b_-0Vkt1s1skpkr2Ikq4ikgkg1_kk32ko___-0B0Fd-1h1h2Ai22E7y2oi5i5bBbB

Cryx Army - 75 / 75 points
[Theme] Slaughter Fleet Raiders

[Skarre 3] Skarre, Admiral of the Black Fleet [+27]
 - Leviathan [16]
 - Leviathan [16]
 - Satyxis Blood Priestess [0(4)]
Axiara Wraithblade [0(6)]
General Gerlak Slaughterborn [0(6)]
Misery Cage [2]
Saxon Orrik [4]
Blighted Trollkin Marauders (max) [15]
Blighted Trollkin Marauders (max) [15]
Bloodgorgers (max) [15]
 - Jussika Bloodtongue [5]
Cephalyx Overlords [8]
Scharde Dirge Seers [6]

Cryx Army - 75 / 75 points
[Theme] Dark Host

[Skarre 1] Pirate Queen Skarre [+28]
 - Inflictor [13]
 - Stalker [8]
 - Stalker [8]
 - Skarlock Thrall [0(4)]
Bane Lord Tartarus [0(6)]
Darragh Wrathe [9]
Orin Midwinter, Rogue Inquisitor [5]
Scrap Thrall (3) [0(2)]
Bane Knights (max) [15]
Bane Knights (max) [15]
Wraith Engine [15]
Wraith Engine [15]
